config/crd/bases/operator.dash0.com_dash0monitorings.yaml

```diff
@@ -54,12 +54,13 @@ spec:
                   `dash0.com:4317`.
                 type: string
               instrumentWorkloads:
+                default: all
```

Comment:
Note: these changes need to be copied over to the crd in the helm chart, otherwise they are not effective. (I usually just diff this file with the helm crd and copy everything over.)